server/src/test/java/com/objectcomputing/checkins/endtoend/testcases/TC01_Verify_Upload_File_Success_Msg.java

```diff
@@ -18,6 +18,7 @@
  */
```

Comment:
is there a reason these tests are commented out ?

Reply:
That endpoint they are testing is not reachable `checkins.objectcomputing.com/`.

I think we are likely getting rid of these selenium tests in the future, but for now I don't want their test failures stopping builds.
These aren't good tests to decide whether the build is good or not in their current state.